From 00d725d1621a5cfd5152f6f2be83c737ac88408d Mon Sep 17 00:00:00 2001
Message-ID: <00d725d1621a5cfd5152f6f2be83c737ac88408d.1705743628.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 Jan 2024 23:46:24 +0000
Subject: [PATCH 20/38] gnu: go-github-com-protonmail-go-crypto: Move to (gnu
 packages golang-crypto).

* gnu/packages/golang.scm (go-github-com-protonmail-go-crypto): Move
from here ...
* gnu/packages/golang-crypto.scm: ... to here.

Change-Id: I8ab594fcb5b5b7eeec777a177eb7541ed49dba90
---
 gnu/packages/golang-crypto.scm | 31 +++++++++++++++++++++++++++++++
 gnu/packages/golang.scm        | 31 -------------------------------
 2 files changed, 31 insertions(+), 31 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index d1d6122f95..417dbc34f6 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -368,6 +368,37 @@ (define-public go-github-com-quic-go-qtls-go1-20
 QUIC.  For Go 1.20.")
     (license license:expat)))
 
+(define-public go-github-com-protonmail-go-crypto
+  (package
+    (name "go-github-com-protonmail-go-crypto")
+    (version "0.0.0-20220623141421-5afb4c282135")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/ProtonMail/go-crypto")
+             (commit (go-version->git-ref version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "05qxdbn8wdk901z5kw2r3jdrag58nxlcsy0p8xd6rq0d71sw94wy"))))
+    (build-system go-build-system)
+    (arguments
+     (list #:import-path "github.com/ProtonMail/go-crypto"
+           #:tests? #f ; Source-only package.
+           #:phases
+           #~(modify-phases %standard-phases
+               ;; Source-only package.
+               (delete 'build))))
+    (propagated-inputs
+     (list go-golang-org-x-crypto))
+    (home-page "https://github.com/ProtonMail/go-crypto")
+    (synopsis "Fork of x/crypto with up-to-date OpenPGP implementation")
+    (description "This package provides cryptography for Go.  This version of
+the package is a fork that adds a more up-to-date OpenPGP implementation.  It
+is completely backwards compatible with @code{golang.org/x/crypto}, the
+official package.")
+    (license license:bsd-3)))
+
 (define-public go-github-com-refraction-networking-utls
   (package
     (name "go-github-com-refraction-networking-utls")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index b8837b4fcd..4de3d321ed 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -3699,37 +3699,6 @@ (define-public gopls
 editor.")
     (license license:bsd-3)))
 
-(define-public go-github-com-protonmail-go-crypto
-  (package
-    (name "go-github-com-protonmail-go-crypto")
-    (version "0.0.0-20220623141421-5afb4c282135")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/ProtonMail/go-crypto")
-                    (commit (go-version->git-ref version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "05qxdbn8wdk901z5kw2r3jdrag58nxlcsy0p8xd6rq0d71sw94wy"))))
-    (build-system go-build-system)
-    (arguments
-     (list #:import-path "github.com/ProtonMail/go-crypto"
-           #:tests? #f ; Source-only package.
-           #:phases
-           #~(modify-phases %standard-phases
-               ;; Source-only package.
-               (delete 'build))))
-    (propagated-inputs (list go-golang-org-x-crypto))
-    (home-page "https://github.com/ProtonMail/go-crypto")
-    (synopsis "Fork of x/crypto with up-to-date OpenPGP implementation")
-    (description
-     "This package provides cryptography for Go.  This version of the
-package is a fork that adds a more up-to-date OpenPGP implementation.
-It is completely backwards compatible with @code{golang.org/x/crypto},
-the official package.")
-    (license license:bsd-3)))
-
 (define-public go-golang-org-x-net
   (let ((commit "8e0e7d8d38f2b6d21d742845570dde2902d06a1d")
         (revision "0"))
-- 
2.41.0

